app/assets/javascripts/users/show.js.coffee

```diff
@@ -1,289 +1,160 @@
 # Place all the behaviors and hooks related to the users show page here.
-setupCallbacks = () ->
-  ($ "a.contrib-delete-link").on "ajax:success", (ee, data, status, xhr) ->
-    ($ ee.target).closest('tr').hide()
+$(document).ajaxSuccess (event, request, settings) ->
+  query = $(event.target.activeElement)
+  if (query.data('method') is 'delete')
+    query.closest('tr').after('<tr></tr>').hide()
 
-  ($ "a.contrib-delete-link").on "ajax:error", (ee, data, status, xhr) ->
-    # TODO Handle Error
+$(document).ajaxError (event, xhr, settings, error) ->
+  quickFlash(error, 'error')
```

Comment:
@stowellm It's not actually an indentation issue. @asalani93 is correct in that it's what causes the problem, but the issue is that the verify code for iSENSE is hilariously bad at the moment, and I overlooked that. So, I'm going to go fix that, then resubmit these lines.